src/components/EditPost/EditPost.jsx

```diff
@@ -81,68 +85,86 @@ const EditPost = ({
     setPostType(e.target.value.toLowerCase());
   }
 
+  let bottomContentSections = <div className="promptpage__bottom-section">
```

Comment:
I think this should be a const and capitalized to indicate it's a component?

Reply:
Yeah, it was getting overly busy in here, my bad. Extracted it out to a separate component now.